platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -640,13 +640,6 @@ This change impacts:
 
 === Deprecations
 
-[[deprecated_fork_options_java_home]]
```

Comment:
🤔 I think this bit should remain, as it was a concern when upgrading from 8.10.

Reply:
🤔 That would imply that someone is looking at the 9.0 docs while upgrading from say, 8.10 to 8.11.  I suppose that's possible, but I kind of look at this guide as providing info about the things that might affect the _current_ version of the docs that you're looking at, with information about when (in what versions) those changes occurred.  I would expect that if they were upgrading to 8.11, they would look at the 8.11 docs.

Having said that, I could be thinking about the purpose of the upgrade grade in the wrong manner.  I'm not opposed to putting this back.  WDYT?

Reply:
I had been treating those update guides as unmodifiable for old versions.  If you looked at the current (9.0) docs to update 8.10 -> 8.11, you would still want to see this info, since you'll need it to make the upgrade.  Then at some point later, when you're ready to go up to 9.0 you'll look at the info for 9.0 that says this has been removed.  If you're going from 8.10 -> 9.0, you should read every step of the way and note that these 2 entries "cancel out".

Reply:
I suspect that most people just read the current version of docs, and do not expect to need to navigate to the 8.11 docs when upgrading 8.10 -> 8.11.  It's annoying/poor UX for this info to not be present.

Reply:
Why would it need to be present? 

This isn't going to be removed in Gradle 9.0

Reply:
My expectation as a user is that the information needed to upgrade from 8.10 to 8.11 remains constant in versions > 8.11, since version 8.10 and version 8.11 don't change after release.  It's counterintuitive for this information to suddenly change when neither 8.10 or 8.11 have changed.

Reply:
> My expectation as a user is that the information needed to upgrade from 8.10 to 8.11 remains constant in versions > 8.11

I don't think you need this information to upgrade from 8.10 to 8.11.

In any case, it like it's worth waiting for input from others before moving forward

Reply:
Have you considered postponing the decision to 10.0 (`.willBeRemovedInGradle9()` -> `.willBeRemovedInGradle10()`) instead of undeprecating this functionality? TBH, I don't see a solid decision here. I think it would be frustrating for users if it's going to be deprecated, then undeprecated, and then deprecated again. 

If you want to proceed with undeprecation, I think the best solution would be to leave the upgrade entry and add a note explicitly stating that it was undeprecated in 9.0 with an explanation.